chat.delta.desktop.yml

```diff
@@ -0,0 +1,126 @@
+app-id: chat.delta.desktop
+tags:
+  - delta.chat
+  - email
+  - chat
+  - messenger
+  - crypto
+base: io.atom.electron.BaseApp
+base-version: '18.08'
+runtime: org.freedesktop.Platform
+runtime-version: '18.08'
+sdk: org.freedesktop.Sdk
+command: /app/bin/run.sh
```

Comment:
The same comment as Exec on the desktop file.

Reply:
yeah, I see what you mean. The actual executable is currently installed in /app/delta/. That is probably not in the path that flatpak sets up. So we would have to use /app/delta/deltchat-desktop which is hardly any better.
I have the feeling that will need a wrapper for setting up some environment variables. So I'd rather wait for the upstream desktop file to be used.

Reply:
Not a blocker of course, there are apps that do that and were accepted before.